### PR TITLE
Feature/3050 fix onboarding bug

### LINKED ISSUE #3050: Null pointer exception on blueprints screen
Context
- Occurs after the teams load for @MMirandi 

Related:
- http://github.com/pixiebrix/pixiebrix-extension/blob/4a89a7e204ddcea0ea1bccb64effcdd64b564140/src/options/pages/blueprints/BlueprintsView.tsx#L67-L67
- https://rollbar.com/pixiebrix/pixiebrix/items/3118/

### PR DESCRIPTION
**Pull request by mnholtz**
Fixes #3050

Was unable to replicate this error, but this hopefully fixes the cause of the bug. 

Additionally, adds RTK tag invalidation so that the blueprints page is updated upon brick creation and removal in the workshop.

### CODE REVIEW

File: src/options/pages/blueprints/onboardingView/useOnboarding.tsx [hunk 1]
```diff
@@ -66,6 +70,7 @@ function useOnboarding(): {
 
   return {
     onboardingType,
+    onboardingFilter: (filters ?? [])[0]?.value?.toLowerCase(),
```
Comment (mnholtz) [inline src/options/pages/blueprints/onboardingView/useOnboarding.tsx:73]: any suggestions for a better way to write this? I'm struggling with a good way to refactor 😬

Comment (twschiller) [reply src/options/pages/blueprints/onboardingView/useOnboarding.tsx:73]: See the array index syntax https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#syntax

`filters?.[0]?.value?.toLowerCase()`

File: src/options/pages/blueprints/useInstallableViewItems.tsx [hunk 2]
```diff
@@ -95,17 +98,19 @@ function useInstallableViewItems(
...
         sharing: {
           packageId: getPackageId(installable),
-          source: getSharingType(installable, organizations ?? [], scope),
+          source: getSharingType(installable, organizations.data ?? [], scope),
         },
         updatedAt: getUpdatedAt(installable),
-        status: isActive(installable) ? "Active" : "Inactive",
+        status: isActive(installable)
+          ? "Active"
+          : ("Inactive" as InstallableStatus),
```
Comment (twschiller) [inline src/options/pages/blueprints/useInstallableViewItems.tsx:113]: What is this cast for?

**Review by twschiller — APPROVED** [review]
@mnholtz See comment on using ?.[0] and also on the `as InstallableStatus` and then merge

File: src/options/pages/blueprints/useInstallableViewItems.tsx (hunk 2 above)
Comment (mnholtz) [reply src/options/pages/blueprints/useInstallableViewItems.tsx]: I was getting a type error that said something about "Active" or "Inactive" being type "string" and not type "InstallableStatus"

**Pull request merged**